woocommerce/wc-admin
# Home Screen - migrated orders panel

Partially fixes [5238](https://github.com/woocommerce/woocommerce-admin/issues/5238)

This PR moves the Orders panel to the `Home` screen. In order to do it, an ActivityPanel component is created in the `Home` screen.
A follow-up PR will be created with the changes and additions necessary to finish up the issue. Those changes are the `Manage all orders` button and the empty activity panel for orders.


### Accessibility




### Screenshots
![Screen Capture on 2020-10-22 at 10-43-10](https://user-images.githubusercontent.com/1314156/96880354-756f4680-1453-11eb-99c5-4968a755151f.gif)


### Detailed test instructions:

- Go to the `Home` screen.
- Verify the orders panel is visible.
- Create a few orders (manually or using the [Smooth generator](https://github.com/woocommerce/wc-smooth-generator)).
- Confirm that the orders in progress number is right.
- Verify that the order card looks like it should.
- Verify the buttons and links are working well.



### Changelog Note:
Dev: Home Screen - migrate orders panel
